glue_astronomy/translators/regions.py

```diff
@@ -82,6 +81,8 @@ def to_object(self, subset):
                 return PolygonPixelRegion(PixCoord(roi.vx, roi.vy))
             elif isinstance(roi, CircularROI):
                 return CirclePixelRegion(PixCoord(*roi.get_center()), roi.get_radius())
+            elif isinstance(roi, EllipticalROI):
+                return EllipsePixelRegion(PixCoord(roi.xc, roi.xy), roi.radius_x, roi.radius_y)
```

Comment:
```suggestion
                return EllipsePixelRegion(PixCoord(roi.xc, roi.yc), roi.radius_x, roi.radius_y)
```

Wrong attribute name, works after this change.

Reply:
Good call! Pushed the fix in the latest commit.